src/plugins/kibana/public/settings/sections/indices/add_data_steps/pattern_review_step/__tests__/is_geo_point_object.js

```diff
@@ -0,0 +1,21 @@
+import isGeoPointObject from '../lib/is_geo_point_object';
```

Comment:
`expect().to.be.ok()` tests whether the result is truthy, `isGeoPointObject` returns a boolean value.
These tests should be `expect(...).to.be(true)` / `expect(...).to.be(false)`